public/docs/_examples/router/ts/app/crisis-center/crisis-detail.component.1.ts

```diff
@@ -26,10 +28,44 @@ import { Subscription }           from 'rxjs/Subscription';
   </div>
   `,
   // #enddocregion template
-  styles: ['input {width: 20em}']
+  styles: ['input {width: 20em}'],
+  animations: [
+    trigger('routeAnimation', [
+      state('*',
+        style({
+          opacity: 1,
+          transform: 'translateX(0)'
+        })
+      ),
+      transition('void => *', [
+        style({
+          opacity: 0,
+          transform: 'translateX(-100%)'
+        }),
+        animate('0.2s ease-in')
+      ]),
+      transition('* => void', [
+        animate('0.5s ease-out', style({
+          opacity: 0,
+          transform: 'translateY(100%)'
+        }))
+      ])
+    ])
+  ]
 })
 // #docregion cancel-save
 export class CrisisDetailComponent implements OnInit, OnDestroy {
+  @HostBinding('@routeAnimation') get routeAnimation() {
```

Comment:
I recommend clarifying that the @HostBinding decorator statements go inside the Component class. SImple way to do it would be to show the class in a code snippet below this.